### PR TITLE
Using /resourses/normalize to set all respective isCoreTeam Boolean. …

### PR DESCRIPTION
**Pull request by lazaroalvarenga**
…Create findCoreTeam user controller function to get core team ranking.

### CODE REVIEW

**Comment by lazaroalvarenga** [comment]
#22

**Review by ribeiroevandro — APPROVED** [review]

File: utils/index.js [hunk 1]
```diff
@@ -123,3 +123,9 @@ export const getStyleLog = style => {
 export const analyticsSendCollect = e => {
   require("./analytics").sendCollect(e);
 };
+
+export const isCoreTeam = userId => {
+  const allCoreTeam = config.coreteam.members;
+
+  return allCoreTeam.find(member => member === userId) ? true : false;
```
Comment (omarciovsena) [inline utils/index.js]: `return !!allCoreTeam.find(member => member === userId)`

File: routes/resources.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import config from "config-yml";
+import express from "express";
+
+import userController from "../controllers/user";
+const router = express.Router();
+
+router.get("/normalize", async (req, res) => {
+  const normalizeCoreTeam = await userController.checkCoreTeam();
+  console.log(normalizeCoreTeam);
```
Comment (omarciovsena) [inline routes/resources.js]: Forgot or want this log?

File: controllers/user.js [hunk 3]
```diff
@@ -53,8 +57,10 @@ export const update = async interaction => {
           throw new Error("Error updating user");
         }
         const newScore = doc.score + score;
+        // doc.isCoreTeam =
```
Comment (omarciovsena) [inline controllers/user.js]: I think it would be nice to remove this comment

File: controllers/user.js [hunk 4]
```diff
@@ -43,6 +45,8 @@ export const update = async interaction => {
   const score = calculateScore(interaction);
   const userInfo = await getUserInfo(interaction.user);
 
+  // isCoreTeam(interaction.user);
```
Comment (omarciovsena) [inline controllers/user.js]: I think it would be nice to remove this comment

**Review by omarciovsena — COMMENTED** [review]
I think it would be nice to have the `config.coreteam` configuration file

File: controllers/user.js (hunk 4 above)
Comment (lazaroalvarenga) [reply controllers/user.js]: Yes! Thanks for notify.

File: controllers/user.js (hunk 3 above)
Comment (lazaroalvarenga) [reply controllers/user.js]: Yes! Thanks for notify.

File: routes/resources.js (hunk 2 above)
Comment (lazaroalvarenga) [reply routes/resources.js]: Forgot it 😅

File: utils/index.js (hunk 1 above)
Comment (lazaroalvarenga) [reply utils/index.js]: Sounds good! 👏

**Review by omarciovsena — APPROVED** [review]

**Pull request merged**